frontend/coprs_frontend/coprs/__init__.py

```diff
@@ -38,17 +39,19 @@
     extension_responses=[TeamsResponse]
 )
 
+session = Session(app)
```

Comment:
How is this global used?

Reply:
It's not, sorry, in my projects, I usually declare these flask.app wrappers as variables even if they're not used at all. Can be safely changed to Session(app).

Reply:
I'm fine with this, just wanted to confirm that.